### PR TITLE
Add make targets,scripts & docs for bundle format

### PR DESCRIPTION
**Pull request by nikhil-thomas**
Add make targets for buinding bundle images and index images

Add development docs

Delte obsolete scripts and csv manifests

Signed-off-by: Nikhil Thomas <nikthoma@redhat.com>

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+visible in AllNamespaces.
+
+Filter by Provider `OpenShift Pipelines`
+
+Follow usual OperatorHub workflow to install the operator.
+
+
+
+
+
```
Comment (piyush-garg) [inline docs/development.md:66]: Is it something not showing up here, or by mistake empty lines

Comment (nikhil-thomas) [reply docs/development.md:66]: oops

**Pull request merged**